pyne/serpent.py

```diff
@@ -285,7 +292,18 @@ def parse_dep(depfile, write_py=False, make_mats=True):
 
     # Execute the adjusted file
     dep = {}
-    exec(f, dep, dep)
+    try:
+        exec(f, dep, dep)
+    except ValueError:
+        corrected_f = VOL_serp2_fix(quicksave_f, header)
+
+        # Overwrite the file made before to reflect changes
+        if write_py:
+            with open(new_filename, 'w') as pyfile:
+                pyfile.write(corrected_f)
+
+        exec(corrected_f,dep,dep)
```

Comment:
Mainly PEP8 comments,

Just thinking, to avoid multiplying the code to form the footer, maybe we could have a single method to do it, that you call first without enablign the `vol` addition, and if the exec fails you call it back adding the `vol` into it ?
something like:
```
f_out = header + f + form_footer(add_vol=False)
try():
    exec...
catch(...):
    f_out = header + f + form_footer(add_vol=True)
    exec...
```